admission-webhook/webhook/base/bootstrap-cluster-role-binding.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
It might make sense to separate bootstrap from webhook and then in kubeflow/bootstrap/config add all 3 targets. I've created a branch under my fork so you can see what I'm suggesting
https://github.com/kkasravi/manifests/tree/webhook1/admission

comparison - https://github.com/kubeflow/manifests/compare/master...kkasravi:webhook1?expand=1

Also you could probably add podpreset_crd/base contents to bootstrap.

We do have a 'Best Practices' document that is pending. (PR #56) https://github.com/kubeflow/manifests/pull/56/files#diff-c6b5b5feb0975989b455b5d1dc3e9aed

Reply:
👍